Automattic/jetpack
# Contact info

## WIP Design, Tests




Fixes [#2166](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2166)

#### Changes proposed in this Pull Request:

* Selecting `contact info` block instead of `Email` block on insert.
* Changing font size on child blocks to default `16px`.
* Upping line height on child blocks with `2px` padding.
* Using Rich Text placeholder colors for dark and light modes.
* Pulled `Link address to Google Maps` toggle into block settings on `Address` child block.
* Changing keyboard type for `Email` to email keyboard, `Phone` to phone keypad.
* Restricted available blocks to insert when in `contact info` block to: `phone`, `address`, `email`, `separator`, `heading`, and `spacer`.

#### Jetpack product discussion
Currently changing behaviours for `contact-info` for both web and native:
  * Select `contact info` block on insert instead of `email`.
  * Restricting choices to above choices when inserting child blocks.

Waiting on further dev input to decide on whether to keep web changes or not.

#### Does this pull request change what data or activity we track or use?
N/A

#### Testing instructions:






1. Insert `contact info` block. Ensure that the block itself is selected instead of any of the children blocks.
2. Ensure that placeholder text for child blocks are correct color.
3. Type in text into `email` field. Ensure that text is correct color.
4. Repeat for child blocks `phone` and `address`.
5. Select `address` block.
6. Select the settings on `address` block. Ensure that `Link to Google maps` toggle works and settings block looks correct.
7. Switch to dark mode.
8. Repeat steps 2-6 look correct in dark mode.

#### Proposed changelog entry for your changes:

* Changing contact info block behaviour and design to match other blocks.
